solidity/contracts/ConnextVestingWallet.sol

```diff
@@ -122,4 +124,13 @@ contract ConnextVestingWallet is VestingWalletWithCliff, Ownable2Step {
       _token.transfer(_to, _amount);
     }
   }
+
+  /**
+   * --- Claim ---
+   * @notice      Claim tokens from Llama Vesting contract
+   * @dev         This func is needed because only the recipients can claim
+   */
+  function claim(address _llamaVestAddress) external onlyOwner {
+    IVestingEscrowSimple(_llamaVestAddress).claim(address(this));
+  }
```

Comment:
hmm ok, i wonder 2 things, i'll ask the ape about:
- should we make this method `onlyOwner`? since is only claiming from llama
- shouldn't we make the claim inside `release` method? (adds complexity imo, so i'd leave it separate)

Reply:
wdyt @skeletor-spaceman ?

Reply:
i think we should make it permisionless, because is just withdrawing the streamed tokens to this address, and then we keep the `onlyOwner` availability to withdraw the unlocked tokens.

makes sense to make it separate from the withdraw method, not to fall on any unexpected revert and block it all, separation of concerns, but please make this method permisionless 🙏  (so we can execute it with any wallet)